js/load.js

```diff
@@ -0,0 +1,272 @@
+"use strict";
+/******************/
+// sample of how to use streamers data
+$(() => {
+    // callback once streamers data is
+    // loaded from the google sheet
+    loadStreamerData(streamers => {
+        console.log(streamers);
+        // see the structure of streamers
+        // data at 'data/01-07-20.json`
+
+        // do stuff here
+    });
+});
+/******************/
+
+
+const ParsingMethods = {
+    none: (val) => {
+        return val;
+    },
+    stringArray: (val) => {
+        return val != undefined ? val.toString().replace(/[^a-z0-9+]+/gi, ' ').split(' ') : [];
+    },
+    lowercaseValue: (val) => {
+        return (val != undefined) ? val.toLowerCase() : val;
+    },
+    bool: (val) => {
+        return ["Y", "Cam", "Yes"].includes(val);
+    }
+};
+
+const STREAMER_DATA_FIELDS = {
+    'Year/Date of Birth': {
+        name: 'DOB', // the column name in the google sheet
+        uiLabel: 'Birth year', // what we use to display in labels in the UI
+        parsingMethod: ParsingMethods.none // how to parse the data from the cell value
+    },
```

Comment:
I see, this was an over-site on my part.

Could something like this be used?
```
/**
 * Finds the UI label for a given property name
 * i.e. "alcohol" -> "Drinking"
 **/
const getUILabel = (name) => {
    for (var e in STREAMER_DATA_FIELDS) {
        if (STREAMER_DATA_FIELDS[e].name === name) {
            return STREAMER_DATA_FIELDS[e].uiLabel;
        }
    }
};
```

Reply:
Sure, that seems like it would be perfect. I wonder if we'd ever encounter any performance problems from this though, if somehow the datatable increased horizontally, but I'm not sure what else we may be adding. For now it's fine, and it's not really a big problem just didn't want us to leave those incorrect commenting like that and forget about it later.